frontend/e2e/$lang+/_public/apply/adult/_adult-flow.spec.ts

```diff
@@ -0,0 +1,234 @@
+import type { Page } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+
+// Reusable funtion to check empty input
+async function hasError(page: Page, error: string) {
+  await page.getByRole('button', { name: 'Continue' }).click();
+  await expect(page.getByRole('link', { name: error })).toBeVisible();
+}
+
+// Reusable function to fill out date of birth
+async function fillOutDOB(page: Page, year: string) {
```

Comment:
Remove the first line action click `Continue` button should be for the previous page step. Ensure you calculate the year to use if you are targeting a specific age group. This test might work today but might fail next year because the dob land in a different age group.